vpr/src/route/bucket.h

```diff
@@ -257,18 +259,44 @@ class Bucket : public HeapInterface {
 
     size_t seed_; /* Seed for fast_rand, should be non-zero */
 
-    BucketItem** heap_; /* Buckets for linked lists*/
-    size_t heap_size_;  /* Number of buckets */
-    size_t heap_head_;  /* First non-empty bucket */
-    size_t heap_tail_;  /* Last non-empty bucket */
-    float conv_factor_; /* Cost bucket scaling factor */
+    BucketItem** heap_;      /* Buckets for linked lists*/
+    size_t heap_size_;       /* Number of buckets */
+    size_t heap_head_;       /* First non-empty bucket */
+    size_t heap_tail_;       /* Last non-empty bucket */
+    float conv_factor_;      /* Cost bucket scaling factor */
```

Comment:
Can you explain this more?  Do you multiply cost by conv_factor to find the bucket?